simulator/src/main/java/simblock/simulator/Main.java

```diff
@@ -18,6 +18,7 @@
 
 
 import simblock.block.Block;
+import simblock.node.Farmer;
```

Comment:
mostly auto reformatting by intelliJ and a few changes for parametrizing between PoW and ChiaPoSpace